{{cookiecutter.project_folder}}/pyproject.toml

```diff
@@ -4,31 +4,31 @@ version = "{{ cookiecutter.version }}"
 description = "{{ cookiecutter.short_description }}"
 authors = ["{{ cookiecutter.full_name }} <{{ cookiecutter.email }}>"]
 readme = "README.md"
-packages = [{include =  "{{ cookiecutter.module_name }}", from = "{{ cookiecutter.namespace }}"}]
+package-mode = false
```

Comment:
this is arguable thing, if the idea is to have a project as a dependency of other projects then we may need to keep the original value.
from my perspective current project is an independent project

Reply:
I agree. I don't see this project as a dependency for other projects